### ISSUE TITLE
Changes in the voice

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6787]
**Preferred term label:**
Changes in the voice

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7528943/

**Comment by pnrobinson** [comment]
This is very close or synonymous with Dysphonia HP:0001618 (https://hpo.jax.org/app/browse/term/HP:0001618)

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK565881/

**Comment by pnrobinson** [comment]
added as synonym ti Dysphonia
